Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -401,6 +402,18 @@ extension Datadog {
                 return self
             }
 
+            /// Sets the custom mapper for `logEvent`. This can be used to modify logs before they are send to Datadog.
+            /// - Parameter mapper: the closure taking `logEvent` as input and expecting `logEvent` as output.
+            /// The implementation should obtain a mutable version of the `logEvent`, modify it and return it.
```

Comment:
```suggestion
            /// Sets the custom mapper for `LogEvent`. This can be used to modify logs before they are send to Datadog.
            /// - Parameter mapper: the closure taking `LogEvent` as input and expecting `LogEvent` as output.
            /// The implementation should obtain a mutable version of the `LogEvent`, modify it and return it.
```